modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/init.jsp

```diff
@@ -87,7 +91,8 @@ page import="com.liferay.portal.kernel.util.ParamUtil" %><%@
 page import="com.liferay.portal.kernel.util.PortalUtil" %><%@
 page import="com.liferay.portal.kernel.util.StringUtil" %><%@
 page import="com.liferay.portal.kernel.util.Validator" %><%@
-page import="com.liferay.portal.kernel.util.WebKeys" %>
+page import="com.liferay.portal.kernel.util.WebKeys" %><%@
+page import="com.liferay.portlet.documentlibrary.lar.FileEntryUtil" %>
```

Comment:
this looks wrong, can you check the sorting?

Reply:
It was right since portlet comes after portal in sorting. But I've removed this import